### PR TITLE
chore(fido-device-onboard.spec): update to 0.4.5 latest

### PR DESCRIPTION
**Pull request by runcom**
Signed-off-by: Antonio Murdaca <runcom@linux.com>

### CODE REVIEW

File: fido-device-onboard.spec [hunk 1]
```diff
@@ -66,6 +69,7 @@ install -D -m 0755 -t %{buildroot}%{dracutlibdir}/modules.d/52fdo dracut/52fdo/m
 
 %package -n fdo-init
 Summary: dracut module for device initialization
+Requires: openssl-devel >= 3.0.1-12
```
Comment (puiterwijk) [inline fido-device-onboard.spec:72]: `openssl-devel` is a `BuildRequires`. You want `openssl-libs` for runtime dependencies.

Comment (runcom) [reply fido-device-onboard.spec]: fixed

**Comment by runcom** [comment]
updated to the latest and greatest

**Review by puiterwijk — APPROVED** [review]
Thanks!

**Comment by puiterwijk** [comment]
@Mergifyio update

**Pull request merged**